### PR TITLE
Add a new storyline about Ghostbusters.

### PR DESCRIPTION
**Pull request by wwarne**

### CODE REVIEW

**Comment by wwarne** [comment]
Please review this, @casey-collab

File: russian/call-ghostbusters/ghostbusters.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿Под звуки знаменитой песни, наигрывающей в голове, вы набираете 8-800-GHOSTBUSTERS и пытаетесь объяснить ситуацию.
+
+Похоже на проделки Зефирного человека! Охотники уже в пути! - слышите вы и облегчённо кладёте трубку.
+
+Что же делать теперь?
+
+[Откроете окно](../window/window.md)
+[Вернётесь ко сну.](../sleep/zefir.md)
```
Comment (casey-collab) [inline russian/call-ghostbusters/ghostbusters.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wwarne** [comment]
Thanks.

**Comment by wwarne** [comment]
Thanks @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smile:

**Pull request merged**